packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/dashboard/dashboard.controller.js

```diff
@@ -22,10 +23,15 @@ export default /* @ngInject */ function TelecomTelephonyBillingAccountDashboardC
   const self = this;
 
   function isExpired() {
-    if (self.group) {
-      return self.group.status === 'expired';
-    }
-    return true;
+    return self.group ? self.group.status === 'expired' : false;
```

Comment:
```suggestion
    return self.group?.status === 'expired';
```

Reply:
Ticket has been marked as `qualtity check: ok` before submitting your suggestion.

Since it's not marked as a *Request changes* so I will merge this Pull Request but it can be taken in account into a separate Pull Request.

Thank you!